node/send_receive/tslint.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
how much of this is default settings? ideally we'd have a file that contains only the "exceptions" or custom rules that we desire